promscale/installation/kubernetes.md

```diff
@@ -150,15 +150,15 @@ can provide the database URI, or specify connection parameters.
 ## Install Promscale with a manifest file
 This section includes instructions to install the Promscale Connector using a
 manifest file. To deploy TimescaleDB on Kubernetes use
-[helm charts][timescaledb-install-helm] instead. Alternatively, you can
-[install TimescaleDB on a host][timescaledb-host-install].
+[helm charts](#installing-the-timescaledb-helm-chart) instead. Alternatively, you can
+[install TimescaleDB on a host](#install-timescaledb).
```

Comment:
Why did you change these?

Reply:
Because for some reason those links were broken https://github.com/timescale/docs/runs/7135066225?check_suite_focus=true 

I went to the official docs [here](https://docs.timescale.com/promscale/latest/installation/kubernetes/#install-promscale-with-helm) to get the correct anchors to make the Github action pass. 

Do you want me to revert these 2 changes?

Reply:
No, it's all good! I was just curious, but it seems that doing anchors as reference style isn't working.

Reply:
I'm going a little blind with this. In the `latest` branch I get a 404 and without fixing those 2 links which I really don't know where they are pointing to:

I switched them to anchors because I thought it would work, because it's what works at https://docs.timescale.com/promscale/latest/installation/kubernetes/#installing-the-timescaledb-helm-chart clearly that didn't worked very good 😓 

Can you help me set up the correct links for this page @Loquacity ? 🥺

Reply:
Yeah, the way anchors work in our build system is a little janky, I'm sorry. I'll check this out and take a look for you.

Reply:
Both those links seem to be working fine in the live site as they are, can you confirm? I'm not sure we need to change anything at all here.

Reply:
Yeah, the live site for me it works correct. The issue was the CI on this PR, and the problem was with existing code, not with my changes. 

I just updated the branch. With your changes is all green now, so I'll request a re-review. 

Thanks so much for your help :)